charts/ditto/values.yaml

```diff
@@ -709,7 +709,7 @@ thingsSearch:
 mongodb:
   ## enabled controls usage of dependency chart
   enabled: true
-  fullnameOverride: ditto-mongodb
+  # fullnameOverride: ditto-mongodb
```

Comment:
Shouldn't this be called "fullname" so it corresponds to the _helpers.tpl and mongodb-secret.yaml variables?

Reply:
No, the `_helpers.tpl` uses `.Values.fullnameOverride`.